src/_locales/sv/messages.json

```
@@ -54,5 +54,38 @@
   },
   "btnFullScreen": {
     "message": "Visa i helskärm"
+  },
+  "kbdShortcuts": {
+    "message": "Tangentbordsgenvägar"
+  },
+  "kbdMovement": {
+    "message": "Rörelsenycklar"
+  },
+  "kbdFeature": {
+    "message": "Navigationsnycklar för geografiska egenskaper"
+  },
+  "kbdPanUp": {
+    "message": "Panorera upp"
+  },
+  "kbdPanDown": {
+    "message": "Panorera ner"
+  },
+  "kbdPanLeft": {
+    "message": "Panorera till vänster"
+  },
+  "kbdPanRight": {
+    "message": "Panorera till höger"
+  },
+  "kbdPanIncrement": {
+    "message": "Ökning för panorering av kartan"
```

```suggestion
    "message": "Öka panorering"
```

This roughly translates to "The increase for panning of maps". The context is already a map, so no need to mention that IMO.